src/styles/block/clock.scss

```diff
@@ -0,0 +1,46 @@
+@import "../utils/mixin";
+@import "../utils/variables";
+
+.clock {
+
+  display: flex;
+  margin: 0 auto;
+  background-color: #e5e4e2;
+  @include circle($clock-size);
+  justify-content: center;
+  align-items: center;
+  border: 4px solid #98afc7;
+
+  &__center {
+    height: 30px;
+    width: 30px;
+    background-color: #728fce;
+    border-radius: 50%;
+  }
+
+  &__minute {
+    position: absolute;
+    top: 20px ;
+    width: 10px;
+    height: 280px;
+    background-color: #36454f;
+    border-radius: 10px;
+
+    animation-name: movement;
+    animation-duration: 3600s;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
```

Comment:
check the previous comment
it can look like that
<img width="595" alt="image" src="https://user-images.githubusercontent.com/102411130/212853798-8ab861a9-ece0-45db-86f7-8f02bb9d8a8a.png">